src/components/elements/off-canvas-menu.tsx

```diff
@@ -143,438 +136,355 @@ export const OffCanvasMenuWrapper = (props: OffCanvasMenuWrapperPropps) => {
-                                        <NavCard
-                                            aria_label="Options"
-                                            icon={() => (
-                                                <img src={Options} alt="" width="32" height="32" />
-                                            )}
-                                            content={
-                                                <Localize translate_text="Earn fixed payouts by predicting an asset's price movement." />
-                                            }
-                                            title={<Localize translate_text="Options" />}
-                                            onClick={handleArrowClick}
-                                            to="/trade-types/options/"
-                                        />
-                                    </Flex>
-                                )}
-                                <Flex mb="2rem">
-                                    <NavCard
-                                        aria_label="Multipliers"
-                                        icon={() => (
-                                            <img src={Multipliers} alt="" width="32" height="32" />
-                                        )}
-                                        content={
-                                            <Localize translate_text="Combine the upside of CFDs with the simplicity of options." />
-                                        }
-                                        title={<Localize translate_text="Multipliers" />}
-                                        onClick={handleArrowClick}
-                                        to="/trade-types/multiplier/"
-                                    />
-                                </Flex>
-                            </>
-                        )}
-
-                        <Divider m="16px 0" width="100%" height="1px" color="grey-8" />
-
-                        <Text color="grey-5" mb="8px" size="14px">
-                            {<Localize translate_text="Trading platforms" />}
-                        </Text>
-                        <Flex mb="2rem">
-                            <NavCard
-                                aria_label="DMT5"
-                                icon={() => <img src={DMT5} alt="" width="32" height="32" />}
-                                content={
-                                    <Localize translate_text="Trade on Deriv MT5, the all-in-one CFD trading platform." />
-                                }
-                                title={<Localize translate_text="Deriv MT5" />}
-                                onClick={handleArrowClick}
-                                to={props.is_ppc_redirect ? '/landing/dmt5/' : '/dmt5/'}
-                            />
-                        </Flex>
-                        {is_row && (
+    let accordian_array = [
+        {
+            id: 1,
```

Comment:
Add additional information in the `id` it can be a combination of strings and number that is unique enough to be duplicated

Reply:
okay